reference-implementation/README.md

```diff
@@ -14,14 +14,6 @@ You can check the test coverage at any time by running `npm run coverage` in thi
 
 To run all tests (and the lint step), run `npm test` in this folder.
 
-### Legacy tests
-
-The original tests are written using the [tape](https://github.com/substack/tape) framework. They live in the `test` subfolder of this folder.
-
-- To run them, type `npm run legacy-test` in this folder.
-- To run a specific test file, type e.g. `node run-tests.js test/abstract-ops.js`.
-- To run a specific test, change the source file's `test(...)` call to `test.only(...)`, then run `npm test`.
-
 ### Web platform tests
 
 After we started implementing this in browsers, we realized that we should probably have written the tests in standard [web platform tests](https://github.com/w3c/web-platform-tests) format. Fixing this has been an ongoing effort, with major help from various community contributors.
```

Comment:
This is now a historical note, which is probably not relevant to people reading this for the first time. Remove?

Reply:
Weird, I could swear I did; maybe something did not get committed. Thanks for catching it.